ci-operator/step-registry/redhat-lp-chaos/lp/cnv/windows-vm/create/redhat-lp-chaos-lp-cnv-windows-vm-create-commands.sh

```
@@ -0,0 +1,96 @@
+#!/bin/bash
+set -euxo pipefail; shopt -s inherit_errexit
+
+[ -s "${KUBECONFIG}" ]
+
+# Required by benchmark-runner
+( set +x; KUBEADMIN_PASSWORD=$(cat "${SHARED_DIR}/kubeadmin-password") )
```

Shellcheck (0.11.0)

[info] Modification of KUBEADMIN_PASSWORD is local (to subshell caused by (..) group).

(SC2030)